### PR TITLE
Bump R version

### PR DESCRIPTION
**Pull request by johnkerl**
For first `tiledbsoma` release: with R and Python both

### CODE REVIEW

File: apis/r/DESCRIPTION [hunk 1]
```diff
@@ -7,7 +7,7 @@ Description: Interface for working with 'TileDB'-based Stack of Matrices,
     from and export to in-memory formats used by popular toolchains like
     'Seurat', 'Bioconductor', and even 'AnnData' using the companion Python
     package.
-Version: 0.1.3.9006
+Version: 0.1.12
```
Comment (eddelbuettel) [inline apis/r/DESCRIPTION:10]: Can we make it 0.1.3.9007 and then integrate the first 'proof of light' C++ bindings from R?

**Review by eddelbuettel — DISMISSED** [review]

File: apis/r/DESCRIPTION (hunk 1 above)
Comment (johnkerl) [reply apis/r/DESCRIPTION:10]: @eddelbuettel thanks!

This is on the `main-old` branch not `main` -- completely independent of all `main` development involving C++ etc.

Comment (johnkerl) [reply apis/r/DESCRIPTION:10]: `main` is (for now) effectively a `dev` branch

Comment (aaronwolen) [reply apis/r/DESCRIPTION:10]: The motivation here is to sync the python/R api versions. You could bump the version to 0.1.13.9000 in your PR.

**Review by aaronwolen — APPROVED** [review]

File: apis/r/DESCRIPTION (hunk 1 above)
Comment (johnkerl) [reply apis/r/DESCRIPTION:10]: See also
**https://github.com/single-cell-data/TileDB-SOMA/wiki/Developer-resources**

**Review by aaronwolen — CHANGES_REQUESTED** [review]
Can you update `NEWS.md` with the new version and indicate the bump is to sync with python?

**Comment by johnkerl** [comment]
Good call @aaronwolen ! :)

**Comment by johnkerl** [comment]
@aaronwolen 
https://github.com/single-cell-data/TileDB-SOMA/pull/351/commits/e66026e96781c2ebf5b25789ef5f05060e7821d8

File: apis/r/NEWS.md [hunk 2]
```diff
@@ -83,6 +83,16 @@ Effectively, this stores the data on disk in column-major order. However, the `t
 - Added `with_allocation_size_preference()` helper to temporarily set the allocation size preference for testing.
 - Tests were added to verify the internal `dgtmatrix_to_dataframe()` will error out if an input list contains non-layerable matrices.
```
Comment (aaronwolen) [inline apis/r/NEWS.md:85]: Thanks for adding! But this whole block should be moved to the top of `NEWS.md`, and `# tiledbsc (development version)` should be replaced with `# tiledbsoma 0.1.12` (like [so](https://gist.github.com/aaronwolen/4962f6cfc25a362cadd1175390042c4b)).

**Review by aaronwolen — DISMISSED** [review]

File: apis/r/DESCRIPTION (hunk 1 above)
Comment (eddelbuettel) [reply apis/r/DESCRIPTION:10]: >  This is on the main-old branch 

Gee. Likely not the last time I fell into _that_ pit.  Never mind then.

**Review by aaronwolen — APPROVED** [review]
Thanks!

**Pull request merged**